src/test/java/Homework21.java

```diff
@@ -0,0 +1,38 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework21 extends BaseTest{
+    String newPlaylistName = "Aaryav";
+    @Test
+    public void renamePlaylist()throws InterruptedException{
+        String updatedPlaylistMsg = "Updated playlist \"Aaryav.\"";
+
+        provideEmail("akansha.shukla@testpro.io");
+        providePassword("te$t$tudent");
+        clickSubmit();
+        Thread.sleep(2000);
+        doubleClickPlaylist();
+        enterNewPlaylistName();
+        Assert.assertEquals(getRenamePlaylistSuccessMsg(), updatedPlaylistMsg);
```

Comment:
Please, delete `throws InterruptedException`

**Thread. sleep is bad! It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.**

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.